site/docs/v1/tech/client-libraries/index.adoc

```diff
@@ -12,16 +12,20 @@ Client libraries will help you quickly integrate your application with FusionAut
 
 If we are missing a language, open a https://github.com/FusionAuth/fusionauth-issues/issues[GitHub Issue] as a Feature Request if you don't see it already listed as an open feature.
 
+* link:dart[Dart]
 * link:go[Go]
 * link:java[Java]
-* link:javascript[JavaScript]
 * link:netcore[.NET Core]
-* link:node[Node]
 * link:php[PHP]
 * link:python[Python]
 * link:ruby[Ruby]
 * link:typescript[Typescript]
 
+=== Deprecated libraries
+
+* link:javascript[JavaScript] - superseded by link:typescript[Typescript]
+* link:node[Node] - superseded by link:typescript[Typescript]
+
```

Comment:
Hmmm. It seems like the use cases are:
* access FA in the browser (JavaScript)
* access FA in server side JavaScript (Node)
* access FA from either place (typescript) with static typing if you want

I've seen a couple of issues come across that indicate that the typescript client still has some rough edges https://github.com/FusionAuth/fusionauth-typescript-client/issues (29, 28).

So short term I think you're right, I should remove the deprecation markers and move these libraries back up to their normal spots on the nav and the overview page. I was perhaps a bit overzealous, but the pages themselves make it clear that these libraries are going to be phased out at some point.

Long term I'd love to have just one javascript client library page called 'JavaScript' outlining the different use cases, using the typescript library. 

So, short term plan is to roll back my changes in this PR. Long term plan (I'll put a note to revisit in 3 months) is to consolidate all the use cases under with the name JavaScript and illustrate how you can use the typescript library in node, the browser and with typescript. The go/no go decision on this change is whether the typescript library has the rough edges polished off enough to hide the other two libraries.

Does that make sense to you @robotdan ?

Reply:
Yes, this sounds great. 

I think I'm ok with either path as long as we are able to outline each use case and how to access and utilize our client lib for each use case. 

It may be that way want to plan a short sprint or something like that for client lib work, I think between the NetCore and JavaScript variants we have some issues to work through based upon end user feedback.